adapter/mysql/mysql.go

```diff
@@ -27,10 +27,23 @@ type Adapter struct {
 
 var _ rel.Adapter = (*Adapter)(nil)
 
-// Open mysql connection using dsn.
-func Open(dsn string) (*Adapter, error) {
-	var err error
+// New is mysql adapter constructor.
+func New(database *db.DB) *Adapter {
+	return &Adapter{
+		Adapter: &sql.Adapter{
+			Config: &sql.Config{
+				Placeholder:   "?",
+				EscapeChar:    "`",
+				IncrementFunc: incrementFunc,
+				ErrorFunc:     errorFunc,
+			},
+			DB: database,
+		},
+	}
+}
 
+// Open mysql connection using dsn.
+func Open(dsn string) (_ *Adapter, err error) {
```

Comment:
```suggestion
func Open(dsn string) (*Adapter, error) {
```

Since I never use named returns on this package, I think it's better to keep it consistent